app/assets/javascripts/bootstrap_datepicker.js.erb

```diff
@@ -0,0 +1,16 @@
+//= require bootstrap-datepicker
+
+$(document).on('ContentLoad', function () {
```

Comment:
Basically, we could define a custom language `foreman` and then use our default means for translations.
http://bootstrap-datepicker.readthedocs.io/en/latest/i18n.html